unit_testing/cli/test_cli_sct_register_to_template.py

```diff
@@ -24,7 +24,7 @@ def test_sct_register_to_template_backwards_compat(script_runner):
 @pytest.fixture(scope="module")
 def template_lpi(tmp_path_factory):
     """Change orientation of test data template to LPI."""
-    path_out = tmp_path_factory.mktemp("tmp_data")/'template_lpi'  # tmp_path_factory is needed for module scope
+    path_out = str(tmp_path_factory.mktemp("tmp_data")/'template_lpi')  # tmp_path_factory is needed for module scope
```

Comment:
ah yeah... that's what turned me off of pathlib... but i guess it's just a matter of getting used to it 😊

Reply:
Yep... pathlib can definitely be awkward to work with for packages that don't support it.

In this case, though, the issue was more because we're passing an unparsed list of arguments to `argparse`, and `argparse` requires all strings. 

Hrm... actually, now that I type that out, I wonder if this could be fixed on the argparse level.